rust/lancedb/src/connection.rs

```diff
@@ -194,6 +237,56 @@ impl CreateTableBuilder<false, NoData> {
         }
     }
 
+    /// Apply the given write options when writing the initial data
```

Comment:
Can we expand on this to describe situations where this might be needed?  Or maybe keep it private?

Reply:
I guess it's not necessary. I can remove it.